runtime/src/lib.rs

```diff
@@ -253,15 +253,17 @@ impl InstanceFilter<Call> for ProxyType {
                 Call::Session(..) |
 				Call::Utility(..)
             ),
-            ProxyType::Vesting => matches!(c,
-                Call::Staking(..) |
-                Call::Session(..) |
-                Call::Democracy(..) |
-				Call::Council(..) |
-				Call::Elections(..) |
-				Call::Vesting(pallet_vesting::Call::vest(..)) |
-				Call::Vesting(pallet_vesting::Call::vest_other(..))
-            ),
+            ProxyType::NonProxy => !matches!(c,
+                Call::Proxy(pallet_proxy::Call::add_proxy(..)) |
```

Comment:
Can we just simplify this to `Call::Proxy(...)`

Reply:
So basically no operations on proxy account.

Reply:
But doesn't the `restricted` account need the capability to, for example, transfer tokens from the anonymous account? This would be done via the `pallet_proxy::Call::proxy()` or am I confusing something?

Reply:
ah yes the proxy() call was not included, I oversighted it. 👍